aspnetcore/migration/22-to-30.md

```diff
@@ -525,4 +525,4 @@ If you're using features of `Newtonsoft.Json` that aren't supported in `System.T
 
 ## Opt in to runtime compilation
 
-In 3.0, runtime compilation is an opt-in scenario. To enable runtime compilation, see <xref:mvc/views/view-compilation#runtime-compilation>.
+In 3.0, runtime compilation is an opt-in scenario. To enable runtime compilation, see https://docs.microsoft.com/en-us/aspnet/core/mvc/views/view-compilation?view=aspnetcore-3.0#runtime-compilation.
```

Comment:
Is there a better way to do this? Right now the migration notes take you to the 2.2 docs which aren't helpful.

Reply:
This is how I would've handled it.